### PR TITLE
Add loading text for long-running integration install step

### PR DESCRIPTION
**Pull request by ofalvai**
When setting up a new integration for the first time, it takes a few seconds to install the selected integration. This PR adds a label besides the spinner which explains that it takes so long only the first time.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ofalvai, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#4378).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by bramkragten** [comment]
It is actually waiting for the device and area registry to be loaded. This should not take long. When do you experience this? And how many devices and areas do you have?

**Comment by ludeeus** [comment]
For a custom component and venv installs it will also install requirements during that time

**Comment by bramkragten** [comment]
I think that is a different loading element:
https://github.com/home-assistant/home-assistant-polymer/blob/7d9b6ab8bc9f3d06afa966967e87fc4e431f6068/src/dialogs/config-flow/dialog-data-entry-flow.ts#L131

**Comment by ludeeus** [comment]
yeah, probably.
Was just trying to add context to what he/she was trying to "solve" 😄

**Comment by bramkragten** [comment]
Hi @ofalvai,

Thanks for your contribution, but unfortunately I have to close your PR as it has gone stale.

If you want to continue to work on this, reopen or open a new PR!

Thanks.

**Comment by ofalvai** [comment]
Hey! Sorry, I didn't see the previous comments. I'm running Home Assistant on a Raspberry Pi 3 Model B+, and it takes around 5-10 seconds for the integration to load. If I open `htop` during that time, I can see `pip install` processes using the CPU. I'm running HA in a venv though.

**Comment by bramkragten** [comment]
Yes, I know what you mean, but I think you edited the wrong loader. Did you test your code?

**Comment by ofalvai** [comment]
To be honest, I don't remember, I edited multiple files and then reverted some changes. I'll fix it if you think this PR makes sense and has a chance to be merged.

**Comment by bramkragten** [comment]
Yeah it does :-)

**Comment by ofalvai** [comment]
@bramkragten can you re-open this PR? I pushed a commit to my fork, but it doesn't show up in this closed PR.

**Comment by ofalvai** [comment]
Thanks! Can I ask for a review @bramkragten? :)

File: src/dialogs/config-flow/step-flow-loading.ts [hunk 1]
```diff
@@ -5,14 +5,20 @@ import {
...
 @customElement("step-flow-loading")
 class StepFlowLoading extends LitElement {
+  @property() public label?: string;
+
   protected render(): TemplateResult {
     return html`
       <div class="init-spinner">
+        <div>
+          ${this.label || ""}
+        </div>
```
Comment (bramkragten) [inline src/dialogs/config-flow/step-flow-loading.ts]: Is the `div` needed? If so do this:
```ts
${this.label ? html`<div>${this.label}</div>` : ""}
```

File: src/translations/en.json [hunk 2]
```diff
@@ -1463,7 +1463,8 @@
             "external_step": {
               "description": "This step requires you to visit an external website to be completed.",
               "open_site": "Open website"
-            }
+            },
+            "loading_first_time": "Please wait while integration is loaded for the first time"
```
Comment (bramkragten) [inline src/translations/en.json]:
```suggestion
            "loading_first_time": "Please wait while the integration is being installed"
```

**Comment by bramkragten** [comment]
Please remove the issue templates from the PR

**Comment by ofalvai** [comment]
@bramkragten Thanks for the review, fixed all the issues you mentioned. Can you take another look?

**Review by bramkragten — APPROVED** [review]

**Pull request merged**